### ISSUE TITLE
Standardize on API endpoints in examples

### ISSUE THREAD

**Issue by Tyrrrz** [passwordless/docs#45]
Currently, the docs are a bit misaligned on API routes when it comes to user-implemented backend endpoints.

For example, docs for the JS client (frontend) use `/create-token` and `/signin`: https://docs.passwordless.dev/guide/frontend/javascript.html#register

While the docs for the .NET client (backend) use `/create-token` and `/verify-signin`: https://docs.passwordless.dev/guide/backend/dotnet.html

Additionally, docs for the JS client assume that `/create-token` returns the raw token (string) directly: https://docs.passwordless.dev/guide/frontend/javascript.html#register

While the docs for the .NET client return the token as part of a JSON object: https://docs.passwordless.dev/guide/backend/dotnet.html

This inconsistency can lead to silly bugs and it would be nice to standardize on some convention.

**Comment by abergs** [comment]
Agreed. Previously we returned a string but switched to returning an object, so the JS-client docs are just out of date.
